src/main/java/lotto/domain/LottoWinningLevel.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LottoWinningLevel {
+    THREE(5000),
+    FOUR(50000),
+    FIVE(1500000),
+    SIX(2000000000);
```

Comment:
```suggestion
    SIX(2_000_000_000);
```
사소한 팁으로 언더스코어를 통해 가독성을 향상 시킬 수 있습니다.